boards/xtensa/nxp_adsp_imx8x/board.cmake

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-board_set_flasher_ifnset(misc-flasher)
-board_finalize_runner_args(misc-flasher)
+board_set_flasher_ifnset(misc)
+board_finalize_runner_args(misc)
```

Comment:
want to to dedup these in the template while you are at it?

Reply:
@fabiobaltieri sorry, I don't know what you mean

Reply:
There's load of these in `board.cmake`, all other runners have a template file for the `board_set_flasher_ifnset` and `board_finalize_runner_args` pair, that's what I was trying to get rid of with my PR.